lib-ee/emqx_ee_bridge/src/emqx_ee_bridge.app.src

```diff
@@ -1,13 +1,14 @@
 {application, emqx_ee_bridge, [
     {description, "EMQX Enterprise data bridges"},
-    {vsn, "0.1.10"},
+    {vsn, "0.1.11"},
     {registered, []},
     {applications, [
         kernel,
         stdlib,
         emqx_ee_connector,
         telemetry,
-        emqx_bridge_kafka
+        emqx_bridge_kafka,
+        emqx_bridge_pulsar
```

Comment:
It still needs to be declared here so that its dependencies are started (`pulsar`).

I'll double check locally, but this is needed to avoid some crashes during node startup with message like "pulsar:_ is not defined".